src/Tribe/Subscribe/Purchase.php

```diff
@@ -54,23 +61,18 @@ public function connect( $attendee_id, $post_id, $order, $product_id ) {
 			],
 		];
 
-		try {
-			$hubspot = Factory::createWithToken( $access_token, $client );
-			$response = $hubspot->contacts()->createOrUpdate( $contact, $properties );
-		} catch ( Exception $e ) {
-			$message = sprintf( 'Could not update or create a contact with HubSpot, error code: %s', $e->getMessage() );
-			tribe( 'logger' )->log_error( $message, 'HubSpot Contact' );
-
-			return;
+		foreach ( $groups as $group ) {
```

Comment:
Maybe instead of this foreach, you could do:

```suggestion
		$properties = array_merge( $properties, ...$groups );
```

Note: You can't use a positional argument after usage of a `...$var` because PHP will throw errors. BUT this works as intended just magically because array_merge() supports merging any number of arguments you send it.

Reply:
added the spread operator.